src/main.js

```diff
@@ -17,27 +17,16 @@ specific language governing permissions and limitations
 under the License.
 */
 
-try {
-    // Ensure npm install has been run.
-    Object.keys(require('../package').dependencies).forEach(require);
-} catch (e) {
-    console.log('Please run "npm install" from this directory:\n\t' + __dirname); // eslint-disable-line no-path-concat
-    process.exit(2);
-}
+// Verify that npm install was run before importing anything else
+require('./check-npm-install-util');
 
 const co = require('co');
 const optimist = require('optimist');
 
 const executil = require('./executil');
 const apputil = require('./apputil');
 
-function * lazyRequire (name, opt_prop) {
-    if (opt_prop) {
-        yield require(name)[opt_prop];
-    } else {
-        yield require(name);
-    }
-}
+const lazyRequire = require('./lazy-require-util.js');
```

Comment:
Is there a difference between just requiring with or without `.js` at the end?

Reply:
I just removed `.js` for the sake of consistency with the rest of the code. (I remember reading that it is better to require with `.js` at the end, but don't think it is worth breaking the consistency in this case.)